### DISCUSSION TITLE
IAP extension recommendations.

### DISCUSSION THREAD

**Discussion by khdhhde21** [airsdk/Adobe-Runtime-Support#2886] [Q&A] [answered]
Hi. I have an app on Google Play that has been using Milkman Games IAP extension for years. Now I need to update the app, but the extension throws an error during the build, so I guess it's time to move to another extension. 

What is the best IAP extension (free or paid) that supports the latest versions of Android? I just need the most basic functionality. No subscriptions, etc.

Thanks!

**Comment by FliplineStudios** [comment, marked as answer]
We switched to Distriqt's In-App Billing extension after dropping the Milkman ANEs, which works great for us on iOS and Android:
https://airnativeextensions.com/extension/com.distriqt.InAppBilling

**Reply by khdhhde21** [reply]
Thank you! That's what I was looking for!
